model/user.go

```diff
@@ -173,3 +174,56 @@ func GetUserScore(username string) (score UserScore, err error) {
 	`, username).Scan(&score.Score)
 	return
 }
+
+// Friend holds a friend
```

Comment:
`// Friend holds a friend with the associated user and the date they became friends`